src/main/webapp/profLoader.js

```diff
@@ -64,9 +80,9 @@
     const commentList = [];
 
     for (let dataHolder of profData) {
-      courseList.push(dataHolder.term);
+      courseList.push(`${dataHolder.course} - ${dataHolder.term}`);
       perceptionAvgList.push(
-        average(dummyGradeData.concat(dataHolder.perceptionList))
+        average(dummyGradeData.concat(dataHolder.perceptionList)) + 7
```

Comment:
why 7? Just wondering

Reply:
Its just dummy data to make the graphs data look real lol